src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/SecurityConfiguration.kt

```diff
@@ -41,16 +41,24 @@ class SecurityConfiguration : WebSecurityConfigurerAdapter() {
         authorize("/info", permitAll)
         authorize(anyRequest, hasRole("PECS_JPC"))
       }
+      sessionManagement {
+        invalidSessionUrl = authLogoutSuccessUri
+        sessionAuthenticationErrorUrl = authLogoutSuccessUri
+        sessionConcurrency {
+          maximumSessions = 1
```

Comment:
Limits maximum number of concurrent session to one for a given user.  The knock on effect of this is if a user logs in a second time then then their other session will be automatically ended.